api/mod/tests/integration/test_login_router.py

```diff
@@ -3,19 +3,36 @@
 """
 
 from fastapi.testclient import TestClient
+from sqlalchemy.orm import Session
 
+from mod.src.database import db_models
+from mod.src.models import user
 
-def test_login(client: TestClient) -> None:
+
+def test_login(client: TestClient, db: Session, user: user.UserCreate) -> None:
     """Test POST /login"""
+    db_models.User.create(db=db, user=user)
     response = client.post(
         "/login/",
-        json={"username": "AnnotoUser#1337", "password": "test1234"},
+        json={"username": "AnnotoUser#1337", "password": "password"},
```

Comment:
If you really want to change it, then please to something that exists in your database test entries (as before, the sample data was functional during local development / testing)

Reply:
Ok yeah we will make that fit again

Reply:
different username